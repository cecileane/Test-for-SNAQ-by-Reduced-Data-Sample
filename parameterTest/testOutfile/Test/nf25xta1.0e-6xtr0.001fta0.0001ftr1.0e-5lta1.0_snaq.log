optimization of topology, BL and inheritance probabilities using:
 hmax = 1,
 tolerance parameters: ftolRel=1.0e-5, ftolAbs=0.0001,
                       xtolAbs=1.0e-6, xtolRel=0.001.
 max number of failed proposals = 25, liktolAbs = 1.0.
rootname for files: nf25xta1.0e-6xtr0.001fta0.0001ftr1.0e-5lta1.0_snaq
BEGIN: 2 runs on starting tree (2,3,(4,(5,(1,6):0.386):1.19):0.028);

main seed 60103
seed: 60103 for run 1
Sat Oct  1 17:06:32 2016

 BEGIN SNaQ for run 1, seed 60103 and hmax 1 changed starting topology by NNI move

Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 42 (failures: 25) with -loglik=215.7344 and ht_min=[0.42251,2.21154,0.0,1.00294,0.11872]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 6		 5		 1	 |	 NA 		 NA 		 NA 
mvorigin	 9		 9		 0	 |	 0		 0		 --
mvtarget	 10		 10		 0	 |	 0		 0		 --
chdir		 5		 5		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 12		 7		 1	 |	 2		 2		 --
Total		 42		 36		 3	 |	 2		 2		 2
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 1.0

 FINISHED SNaQ for run 1, -loglik of best 215.73440081417294
(4,2,((((5,#H9:::0.42250508507420514):2.211537692780081,6):1.0029423941579496,1):0.11872203127311855,(3)#H9:::0.5774949149257949):0.0);
---------------------
seed: 49437 for run 2
Sat Oct  1 17:06:36 2016

 BEGIN SNaQ for run 2, seed 49437 and hmax 1
Begins heuristic optimization of network------

found best network, now we re-optimize branch lengths and gamma more precisely
STOPPED by number of failures criteria
END optTopLevel: found minimizer topology at step 47 (failures: 25) with -loglik=126.74251 and ht_min=[0.2856,0.51916,0.48942,10.0,1.51213]
PERFORMANCE: total number of moves (proposed, successful, accepted) in general, and to fix gamma=0.0,t=0.0 cases
	--------moves general--------				 --------moves gamma,t------	
move	 Num.Proposed	 Num.Successful	 Num.Accepted	 | Num.Proposed	 Num.Successful	 Num.Accepted
add		 1		 1		 1	 |	 NA 		 NA 		 NA 
mvorigin	 12		 12		 1	 |	 0		 0		 --
mvtarget	 14		 14		 1	 |	 0		 0		 --
chdir		 9		 9		 1	 |	 0		 0		 --
delete		 0		 0		 0	 |	 0		 0		 --
nni		 11		 8		 1	 |	 7		 7		 --
Total		 47		 44		 5	 |	 7		 7		 4
Proportion	 -- 		 1.0		 0.1	 |	 -- 		 1.0		 0.6

 FINISHED SNaQ for run 2, -loglik of best 126.74250981975098
(((2,(1)#H7:::0.7143969263315728):1.5121337398317023,4):0.4894189205683776,3,((6,#H7:::0.2856030736684272):10.0,5):0.5191606989127292);
---------------------

Sat Oct  1 17:06:40 2016
MaxNet is (((2,(1)#H7:::0.7143969263315728):1.5121337398317023,4):0.4894189205683776,3,((6,#H7:::0.2856030736684272):10.0,5):0.5191606989127292); 
with -loglik 126.74250981975098
