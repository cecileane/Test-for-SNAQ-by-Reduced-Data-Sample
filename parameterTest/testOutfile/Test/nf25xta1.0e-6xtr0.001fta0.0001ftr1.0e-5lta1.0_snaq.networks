(((2,(1)#H7:::0.7143969263315728):1.5121337398317023,4):0.4894189205683776,3,((6,#H7:::0.2856030736684272):10.0,5):0.5191606989127292);, with -loglik 126.74250981975098 (best network found, remaining sorted by log-pseudolik; the smaller, the better)
(((2,(#H7:::0.3203226136056501,1):2.681119867615278):0.6567291328984638,4):0.45968167041506647,3,((6)#H7:::0.6796773863943499,5):1.9258803652974277);, with -loglik 128.40089055395052
((4)#H7:::0.6231587879936997,3,((6,((2,#H7:::0.3768412120063003):2.5365600900000573,1):1.1933017031267632):0.22260252714963,5):0.0);, with -loglik 173.0136088288901
((#H7:::0.4084761641759188,4):1.810369435490781,3,((6,((2)#H7:::0.5915238358240812,1):3.281216940665395):0.1526570915419722,5):0.013642022165098843);, with -loglik 192.96757936912132
((3)#H7:::0.6056026268713122,(2,((6,(#H7:::0.3943973731286878,5):7.3814561060053):0.9905242976856972,1):0.09530128130600335):0.0743244401547776,4);, with -loglik 215.18290175623514
(((2,((6,#H7:::0.49515232390013075):4.689645463188562,1):0.0):0.21324180388914804,4):1.1186574387899435,3,(5)#H7:::0.5048476760998692);, with -loglik 232.10549788328512
