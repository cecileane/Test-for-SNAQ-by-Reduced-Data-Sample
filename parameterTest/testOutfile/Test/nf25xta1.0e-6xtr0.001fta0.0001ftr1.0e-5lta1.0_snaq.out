(((2,(1)#H7:::0.7143969263315728):1.5121337398317023,4):0.4894189205683776,3,((6,#H7:::0.2856030736684272):10.0,5):0.5191606989127292); -Ploglik = 126.74250981975098
 Dendroscope: (((2,(1)#H7):1.5121337398317023,4):0.4894189205683776,3,((6,#H7):10.0,5):0.5191606989127292);
 Elapsed time: 7.721132357 seconds in 2 successful runs
-------
List of estimated networks for all runs (sorted by log-pseudolik; the smaller, the better):
 (((2,(1)#H7:::0.7143969263315728):1.5121337398317023,4):0.4894189205683776,3,((6,#H7:::0.2856030736684272):10.0,5):0.5191606989127292);, with -loglik 126.74250981975098
 (4,2,((((5,#H9:::0.42250508507420514):2.211537692780081,6):1.0029423941579496,1):0.11872203127311855,(3)#H9:::0.5774949149257949):0.0);, with -loglik 215.73440081417294
-------
